Base/usr/share/man/man7/kernel_cmdline.md

```diff
@@ -0,0 +1,129 @@
+## Name
+
+Kernel cmdline - boot arguments to control system functionality
+
+## Description
+
+The kernel supports boot parameters to change some of its behavior. This
```

Comment:
Let's call them "arguments" (not parameters) consistently

Reply:
Right, since we use the same word in the title, that's a good idea :)